debian/ubuntu-advantage-tools.postinst

```diff
@@ -246,7 +246,7 @@ add_notice() {
 from uaclient.config import UAConfig
 from uaclient.messages import ${msg_name}
 cfg = UAConfig(root_mode=True)
-cfg.add_notice(label='', description=${msg_name})
+cfg.notice_file.add(label='', description=${msg_name})
```

Comment:
@orndorffgrant I think @lucasmoura meant I should add this.

Reply:
Ah okay! and those changes were already there when I was reviewing - I didn't see a response to his comment so was confused.

Looks good!